spec/ace.md

```diff
@@ -49,6 +52,22 @@ Volumes that are specified in the Pod Manifest are mounted into each of the apps
 For example, say that the worker-backup and reduce-worker both have a `mountPoint` named "work".
 In this case, the executor will bind mount the host's `/opt/tenant1/work` directory into the `path` of each of the matching "work" `mountPoint`s of the two app filesystems.
 
+If the target `path` does not exist in the rendered filesystem, it SHOULD be created, including any missing parent directories.
+If the target `path` is a non-empty directory, its contents SHOULD be discarded (e.g. obscured by the bind mount).
+If the target `path` refers to a file, the ACE SHOULD remove that file and create a directory in its place.
+If multiple targets have overlapping target `path`s (for example, if one is nested within another), the ACE SHOULD consider this an error.
+
+The target `path` directories that the ACE creates SHOULD be owned by UID 0 and GID 0, and have access mode `0755` (`rwxr-xr-xr-x`).
+The ACE implementation MAY provide a method for administrator to specify different permissions on a per-pod basis.
+
+The ACE SHOULD NOT create any paths in the host file system, and MUST consider missing volume source paths an error.
+If the ACE does modify the host file system, it SHOULD be possible to disable this behaviour.
```

Comment:
Any chance this leaves open questions for cache mechanisms and executors that might want to perform lazy source volume set up?